session.go

```diff
@@ -203,174 +196,13 @@ type RESTChannel interface {
 	KickParticipant(ctx context.Context, channelID, userID Snowflake, flags ...Flag) (err error)
 }
 
-// RESTEmoji REST interface for all emoji endpoints
-type RESTEmoji interface {
-	// GetGuildEmoji Returns an emoji object for the given guild and emoji IDs.
-	GetGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) (*Emoji, error)
-
-	// GetGuildEmojis Returns a list of emoji objects for the given guild.
-	GetGuildEmojis(ctx context.Context, id Snowflake, flags ...Flag) ([]*Emoji, error)
-
-	// CreateGuildEmoji Create a new emoji for the guild. Requires the 'MANAGE_EMOJIS' permission.
-	// Returns the new emoji object on success. Fires a Guild Emojis Update Gateway event.
-	CreateGuildEmoji(ctx context.Context, guildID Snowflake, params *CreateGuildEmojiParams, flags ...Flag) (*Emoji, error)
-
-	// UpdateGuildEmoji Modify the given emoji. Requires the 'MANAGE_EMOJIS' permission.
-	// Returns the updated emoji object on success. Fires a Guild Emojis Update Gateway event.
-	UpdateGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) *updateGuildEmojiBuilder
-
-	// DeleteGuildEmoji Delete the given emoji. Requires the 'MANAGE_EMOJIS' permission. Returns 204 No Content on
-	// success. Fires a Guild Emojis Update Gateway event.
-	DeleteGuildEmoji(ctx context.Context, guildID, emojiID Snowflake, flags ...Flag) error
-}
-
 // RESTGuild REST interface for all guild endpoints
 type RESTGuild interface {
```

Comment:
You can remove this, and place the methods directly into the RESTMethods interface.

Reply:
Ah yeah good spot, I'll fix soon.